packages/xstate-wallet/src/logger.ts

```diff
@@ -0,0 +1,20 @@
+import pino from 'pino';
+import {LOG_FILE} from './constants';
+
+// TODO: Is there a better way to determine if we're in a browser context?
+const isBrowserContext = process.env.NODE_ENV === 'test';
```

Comment:
Do you mean `isTestContext`?

Reply:
I actually mean `isBrowserContext`. But the formula is wrong!

```suggestion
const isBrowserContext = process.env.NODE_ENV !== 'test';
```